html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/SponsorEmail.pm

```diff
@@ -40,6 +40,18 @@ has_field 'create_local_account' => (
     },
 );
 
+has_field 'use_mandatory_fields' => (
+    type => 'Toggle',
+    checkbox_value => 'yes',
+    unchecked_value => 'no',
+    label => 'Use mandatory fields',
+    default => pf::Authentication::Source::EmailSource->meta->get_attribute('use_mandatory_fields')->default,
```

Comment:
Shouldn't it be 
default => pf::Authentication::Source::SponsorEmailSource->meta ...